src/index.html

```diff
@@ -17,16 +17,39 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      data-qa="big"
+      class="big-search-form"
+    >
+      <input
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+        class="big-search-bar"
+      />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <img
+        src="/src/images/Search.svg"
+        alt="search image"
```

Comment:
The alt attribute should describe the image content. Instead of 'search image', consider using a more descriptive text like 'search icon'.